### PR TITLE
Added new design rules

### PR DESCRIPTION
**Pull request by m-hilgendorf**
Blocked on internal support for the new rule syntax

### CODE REVIEW

File: manufacturers/rules.stanza [hunk 1]
```diff
@@ -37,11 +45,15 @@ public pcb-rules bay-area-circuits-conservative-rules :
...
   min-drill-diameter = 0.254 ; 10mil
-  min-silkscreen-width = 0.0762 ; 3mil
-  min-pitch-leaded = 0.35 ; guess
-  min-pitch-bga = 0.35 ; guess
+  min-silkscreen-width = 0.0762 ; 3mil ;
+  min-pitch-leaded = 0.35 ; (guess)
+  min-pitch-bga = 0.35 ; (guess)
   max-board-width = 457.2 ; 18in
   max-board-height = 609.6 ; 24in
+  solder-mask-registration = 0.1016 ; 4mil
```
Comment (bpellenard) [inline manufacturers/rules.stanza:53]: be consistent with the order

File: utils/design-vars.stanza [hunk 2]
```diff
@@ -22,12 +22,6 @@ public var PREFERRED-MOUNTING = "smd" ; values in ["smd", "through-hole"]
 public var MIN-CAP-VOLTAGE = 10.0 ; Minimum voltage to allow for capacitors
 
 ; Landpattern variables
-public var SOLDER-MASK-REGISTRATION: Double = 0.15
-public var MIN-SILKSCREEN-TEXT-HEIGHT: Double  = 0.762
-public var MIN-COPPER-FEATURE: Double  = 0.127
-public var MIN-SPACING: Double  = 0.127
```
Comment (bpellenard) [inline utils/design-vars.stanza:28]: It seems it was not used, right?

File: manufacturers/rules.stanza [hunk 3]
```diff
@@ -37,11 +45,15 @@ public pcb-rules bay-area-circuits-conservative-rules :
   min-copper-edge-space = 0.381 ; 10mil outer, but 15mil inner
   min-annular-ring = 0.1524 ; 6mil
   min-drill-diameter = 0.254 ; 10mil
-  min-silkscreen-width = 0.0762 ; 3mil
-  min-pitch-leaded = 0.35 ; guess
-  min-pitch-bga = 0.35 ; guess
+  min-silkscreen-width = 0.0762 ; 3mil ;
```
Comment (PhilippeFerreiraDeSousa) [inline manufacturers/rules.stanza:48]: You can't comment the new line character xD

**Review by PhilippeFerreiraDeSousa — APPROVED** [review]
If it is always the same maybe a function for {clearance(current-rules(), _} could come in handy

File: manufacturers/rules.stanza (hunk 3 above)
Comment (m-hilgendorf) [reply manufacturers/rules.stanza:48]: good catch!

**Review by PhilippeFerreiraDeSousa — APPROVED** [review]
Nice

**Pull request merged**